operator/v1/0000_70_cluster-network-operator_01.crd.yaml-patch

```diff
@@ -0,0 +1,7 @@
+- op: add
+  path: /spec/versions/name=v1/schema/openAPIV3Schema/properties/spec/properties/defaultNetwork/properties/ovnKubernetesConfig/properties/policyAuditConfig/default
+  value:
+    destination: "null"
```

Comment:
no need to duplicate the per-field defaults. Just `{}` as value should be enough.